framework/decode/vulkan_decoder_base.h

```diff
@@ -229,6 +231,8 @@ class VulkanDecoderBase : public ApiDecoder
         HandlePointerDecoder<VkPipeline>                                pPipelines;
     };
     std::unordered_map<format::HandleId, DeferredOperationFunctionCallData> record_deferred_operation_function_call;
+
+    uint64_t block_index_;
```

Comment:
It is not necessary for the `VulkanDecoderBase` to store the current block index